tools/lsp/preview/ui.rs

```diff
@@ -441,6 +442,14 @@ fn color_to_string(color: slint::Color) -> String {
     format!("#{r:02x}{g:02x}{b:02x}{a:02x}")
 }
 
+fn color_to_short_string(color: slint::Color) -> String {
```

Comment:
Do we really need to pass two strings that differ in the leading '#'? I am sure we can add that on the slint side as well.

This is worth fixing before merge to me, but feel free to disagree and merge anyway.